README.md

```diff
@@ -206,6 +203,7 @@ It returns List<[Barcode]()>
 ```
 final result = await barcodeScanner.processImage(inputImage);
 ```
+To know more see this [example](example/lib/VisionDetectorViews/barcode_scanner_view.dart)
```

Comment:
Duplicated link

Reply:
Yes, I just noticed sending a new PR for it now.